src/main/scala/ai/privado/exporter/HttpConnectionMetadataExporter.scala

```diff
@@ -53,19 +53,19 @@ class HttpConnectionMetadataExporter(cpg: Cpg, ruleCache: RuleCache) {
   def getLiteralsFromLanguageFiles: List[String] = {
     val egressLiterals = cpg
       .literal(URL_PATH_WITH_VARIABLE_SYMBOLS)
-      .filter(node => node.code.replaceAll(NON_ALPHANUMERIC, "") != "")
+      .filter(node => ALPHABET.r.findFirstIn(node.code).isDefined)
```

Comment:
Optionally we can also write this as `.where(_.code(ALPHABET))`
[Not a blocker]

Reply:
I will make it short in follow up PR for sure. Thanks for tip

Reply:
@khemrajrathore It seems that to make it work for `_.code(<regex>)`, we have to write a regex that allows other characters as well, such as `(.*)[a-zA-Z](.*)`. Therefore, I believe findFirstIn would be better suited here.